### ISSUE TITLE
Syntax bikeshedding: package:@std/<name>

### ISSUE THREAD

**Issue by littledan** [drufball/layered-apis#30]
`@namespace/module` is a frequently used convention in the JS ecosystem. Could we use it for layered APIs?

As an import specifier, it's not "taken" (i.e., it  produces failure in [the HTML module specifier resolution algorithm](https://html.spec.whatwg.org/#resolve-a-module-specifier), as we still have time to tweak details in package-name-map, so we could make it "just work":

```js
from "@std/foo" import { bar }
```

As a URL, we'd probably want the `package:` prefix, as `@std/name` may already be interpreted as a relative URL. The `package:` prefix is already [proposed](https://github.com/domenic/package-name-maps/issues/23), as it comes up for mapped packages in exactly the same way. A script tag to use a built-in module would look like this:

```js
<script type="module" src="package:@std/virtual-scroller"></script>
```

This strategy might require a slight tweak to the [fallback alternative based on package-name-map](https://github.com/drufball/layered-apis/issues/10): `@std/<name>` would be permitted as a key in the package name map, but it could also be provided as a value of a mapping.

**Comment by ljharb** [comment]
For any namespaced choice, we'd want to claim it as a username on npm to prevent collisions (it's not a convention, the `@namespace` always represents a concrete user on npmjs.com). For https://npmjs.com/~std specifically, we'd need to ask the current owner if they're willing to hand it over, and if not, pick a different name.

**Comment by devsnek** [comment]
that would be @jdalton

**Comment by littledan** [comment]
Agreed that, if we use this syntax, we should do this according to npm and not just snatch it with a change in interpretation on "our end". If we choose something that really evokes the web platform/native built-in modules, then [npm policy](https://docs.npmjs.com/misc/disputes) may help us with the transfer, but definitely what you say is the first step. It's unclear to me how "std" would be interpreted with respect to the policy if we wanted to push that way.

**Comment by obedm503** [comment]
what if instead of a `@namespace` to precede standard modules, we dropped the string identifier and made it syntax
```js
// instead of
import { foo } from "@std/foo";

// use
import { foo } from foo;
```

since these module identifiers are supposed to be urls that the broswer can fetch and execute, quoted/string module name doesn't make sense for something that's supposed to be part of the "standard" library
